Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -12145,27 +12125,26 @@
     "Context": {
       "ReferenceDateTime": "2019-01-31T00:00:00"
     },
-    "Comment": "DOT not matching",
-    "NotSupported": "dotnet, javascript, python, java",
+    "NotSupported": "javascript, python, java",
     "Results": [
       {
-        "Text": "entre 30 Jul. y 30 Sept.",
+        "Text": "entre 30 jul. y 30 sept.",
         "Start": 15,
         "End": 38,
-        "TypeName": "datetimeV2.timerange",
+        "TypeName": "datetimeV2.daterange",
         "Resolution": {
           "values": [
             {
-              "timex": "(T09:30,T19:30,PT10H)",
```

Comment:
Can you add a case for the original meaning here too (time range)?

Reply:
ok, will do